omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -61,6 +61,8 @@
 
 # The fully qualified domain name. Will use the node's fqdn if nothing is
 # specified.
+# node['hostname'] can allow multiple hostname separated by coma.
+# eg. node['hostname.example1.com', hostname.example2.com]
```

Comment:
No need of this comment as we dont need to change the specifications for hostname attribute.